assets/admin/editor-wizard/steps/lesson-details-step.js

```diff
@@ -4,14 +4,25 @@
 const LessonDetailsStep = () => {
 	// TODO Implement this.
 	return (
-		<div>
-			<div>PENDING TO IMPLEMENT</div>
+		<div className="sensei-editor-wizard-modal__columns">
+			<div className="sensei-editor-wizard-modal__content">
+				<h1>Lesson Details Step</h1>
+				<div>PENDING TO IMPLEMENT</div>
+			</div>
+			<div className="sensei-editor-wizard-modal__illustration">
+				<img
+					src={
+						window.sensei.pluginUrl +
+						'assets/dist/images/sensei-pro-upsell.png'
+					}
+					alt="PENDING TO IMPLEMENT, BUT HERE TO SHOW IT WORKING"
+					className="sensei-editor-wizard-modal-upsell__image"
```

Comment:
I think we should use a more generic name here, right?
Maybe `sensei-editor-wizard-modal__illustration-image`?

Reply:
Changed in [fa0760a](https://github.com/Automattic/sensei/pull/5169/commits/fa0760a72c368db4a30903876b0d271f334a509a). Thanks!